client/src/ivis/AnimationControls.js

```diff
@@ -0,0 +1,674 @@
+    static propTypes = {
+        animationControl: PropTypes.object.isRequired,
+        animationStatus: PropTypes.object.isRequired,
+
+        enabled: PropTypes.bool,
+        jumpFactor: PropTypes.number,
+
+        className: PropTypes.string,
+    }
+
+    static defaultProps = {
+        jumpFactor: 0.01,
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.jumpBound = ::this.jump;
+    }
+
+    jump() {
+        const intvAbs = this.getIntervalAbsolute();
+        const jumpDistance = (intvAbs.to.valueOf() - intvAbs.from.valueOf()) * this.props.jumpFactor;
+
+        this.props.animationControl.jumpForward(jumpDistance);
+    }
+
+    render() {
+        return (
+            <Button
+                title={"Jump froward"}
+                icon={"step-forward"}
+                className={styles.mediaButton + " " + (this.props.className || "")}
+
+                type={"button"}
+                onClickAsync={this.props.animationControl.jumpForward && this.jumpBound}
+                disabled={!this.props.animationControl.jumpForward && this.props.enabled}
+            />
+        );
+    }
+}
+
+@withComponentMixins([withAnimationControl, intervalAccessMixin()])
+class JumpBackwardButton extends Component {
+    static propTypes = {
+        animationControl: PropTypes.object.isRequired,
+        animationStatus: PropTypes.object.isRequired,
+
+        enabled: PropTypes.bool,
+        jumpFactor: PropTypes.number,
+
+        className: PropTypes.string,
+    }
+
+    static defaultProps = {
+        jumpFactor: 0.01,
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.jumpBound = ::this.jump;
+    }
+
+    jump() {
+        const intvAbs = this.getIntervalAbsolute();
+        const jumpDistance = (intvAbs.to.valueOf() - intvAbs.from.valueOf()) * this.props.jumpFactor;
+
+        this.props.animationControl.jumpBackward(jumpDistance);
+    }
+
+    render() {
+        return (
+            <Button
+                title={"Jump backward"}
+                icon={"step-backward"}
+                className={styles.mediaButton + " " + (this.props.className || "")}
+
+                type={"button"}
+                onClickAsync={this.props.animationControl.jumpBackward && this.jumpBound}
+                disabled={!this.props.animationControl.jumpBackward || !this.props.enabled}
+            />
+        );
+    }
+}
+
+@withComponentMixins([withAnimationControl])
+class ChangeSpeedDropdown extends Component {
+    static propTypes = {
+        animationControl: PropTypes.object.isRequired,
+        animationStatus: PropTypes.object.isRequired,
+
+        enabled: PropTypes.bool,
+        steps: PropTypes.arrayOf(PropTypes.number),
+
+        factorFormat: PropTypes.func,
+        classNames: PropTypes.object,
+    }
+
+    static defaultProps = {
+        classNames: {},
+        steps: defaultPlaybackSpeedSteps,
+        factorFormat: (f) => format("~s")(f),
```

Comment:
Is it possible to add a nicer default formatter? The time in seconds isn't the best human readable.
![image](https://user-images.githubusercontent.com/9487592/125771559-aecfd7e3-0b44-41eb-b1cf-8b532edca867.png)